### PR TITLE
[Toolbar] Fix QuickMenu button on BottomToolbar

### PR DESCRIPTION
**Pull request by codeimpl**
With the new implemenation of the QuickMenu button, it was decided to
pin it to the bottom toolbar. So, this patch have been modified the
following features.
  1. Migration the existing old user's QuickMenu button to the 6th
position
  2. Fixed the position of the 6th button

Refs: #1011, #1012

### CODE REVIEW

File: src/org/triple/banana/toolbar/SelectedToolbarButtonAdapter.java [hunk 1]
```diff
@@ -187,7 +187,9 @@ public void setButtonList(ArrayList<ToolbarButtonItem> mButtonList) {
     public int getItemCount() {
         if (mButtonList == null) return 0;
 
-        return mButtonList.size();
+        // The total of the button list is six, but QuickMenu button should be fixed at 6th.
+        // So it should exclude the button lists.
```
Comment (romandev) [inline src/org/triple/banana/toolbar/SelectedToolbarButtonAdapter.java:191]: The number of the toolbar buttons should be 6, but the 6th button should be the quick menu button. So, we need to exclude it from the total button count.

**Review by romandev — COMMENTED** [review]
lgtm with nit

File: src/org/triple/banana/toolbar/SelectedToolbarButtonAdapter.java (hunk 1 above)
Comment (codeimpl) [reply src/org/triple/banana/toolbar/SelectedToolbarButtonAdapter.java]: done. thanks!

**Review by romandev — APPROVED** [review]
lgtm

**Pull request merged**